README.md

```diff
@@ -38,6 +38,44 @@ To be able to work with both of these formats through some sort of uniform inter
 
 > **Note:** The information above is *simplified* for the context of the Metamorphoses library. As noted in the introduction, API Blueprint AST is about to be slowly replaced by [API Description Parse Result Namespace](https://github.com/refractproject/refract-spec/blob/master/namespaces/parse-result.md) as the parser output. Once [Fury](https://github.com/apiaryio/fury.js) has adapters to both *API Blueprint* and the *legacy Apiary Blueprint*, producing *API Description Parse Result Namespace* for both, and once *Apiary* starts to use the *API Description Parse Result Namespace* exclusively, this library becomes redundant.
 
+## Interface
+
+```javascript
+var metamorphoses = require('metamorphoses');
+
+
+// Blueprint source
+var source = '# Sample API...';
+
+
+// Select adapter by mime type:
+// -   API Blueprint AST: 'application/vnd.apiblueprint.ast'
+// -   Legacy Apiary Blueprint AST: 'application/vnd.legacyblueprint.ast'
+var mimeType = 'application/vnd.apiblueprint.ast';
+var adapter = metamorphoses.createAdapter(mimeType);
+
+
+// You can also import the adapter directly:
+// -   API Blueprint AST: apiBlueprintAdapter
+// -   Legacy Apiary Blueprint AST: apiaryBlueprintAdapter
+var adapter = metamorphoses.apiBlueprintAdapter;
+
+
+parse(source, ..., function (err, result) {
```

Comment:
Not really clear where this `parse` method is coming from. I'm guessing `protagonist`.